apps/webhook/src/config/index.ts

```diff
@@ -0,0 +1,42 @@
+import * as dotenv from 'dotenv';
+import * as envalid from 'envalid';
+import { str, port } from 'envalid';
+
+dotenv.config();
+
+let path;
+
+switch (process.env.NODE_ENV) {
+  case 'prod':
+    path = `${__dirname}/../.env.production`;
+    break;
+  case 'test':
+    path = `${__dirname}/../.env.test`;
+    break;
+  case 'ci':
+    path = `${__dirname}/../.env.ci`;
+    break;
+  case 'local':
+    path = `${__dirname}/../.env`;
+    break;
```

Comment:
So this is just code I have moved from api to application generic and would not like to do fixes in it right now for this PR, because it is already massive.